pkg/virtctl/expose/expose.go

```diff
@@ -37,7 +37,7 @@ var strServiceType string
 var portName string
 var namespace string
 
-// generate a new "expose" command
+// NewExposeCommand generate a new "expose" command
```

Comment:
Nit: "generate" should now be "generates"

Reply:
fixed